modules/core/shared/src/main/scala/lucuma/core/model/CatalogInfo.scala

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016-2021 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model
+
+import cats._
+import cats.syntax.all._
+import eu.timepit.refined._
+import eu.timepit.refined.cats._
+import eu.timepit.refined.collection.NonEmpty
+import eu.timepit.refined.types.string.NonEmptyString
+import lucuma.core.enum.CatalogName
+import monocle.Focus
+import monocle.Lens
+
+final case class CatalogInfo(
+  catalog:    CatalogName,
+  id:         NonEmptyString,
+  objectType: Option[NonEmptyString]
```

Comment:
I think this is good (the `Option`).  The `id` was added I believe to track guide star catalog ids and the `objectType`, I see now, was mostly inspired by Simbad results.  I'm not sure what `objectType` information we can expect from guide star catalogs so it's good that it is optional.